app/reactiveLayer/behaviorManager/conf/default.ini

```diff
@@ -2,7 +2,7 @@ use_ears        false
 
 [BEHAVIORS]
 
-behaviors	(tagging, pointing, moveObject, ask, followingOrder, recognitionOrder)
+behaviors	(tagging, pointing, moveObject)
 ttsOptions      "iCub"
 
 [followingOrder]
```

Comment:
I added `recognitionOrder` to the config file. 
`FollowingOrder` is not needed anymore. It was not used during the WYSIWYD review, already. 
I'm commiting the related changes.